docroot/themes/humsci/humsci_basic/src/scss/tools/_mixins.links.scss

```diff
@@ -40,6 +40,11 @@
     border-bottom-style: solid;
     @include hb-global-color('border-bottom-color', 'gray-medium');
     @include hb-link--hover-style('tertiary-highlight', 'tertiary-highlight-darken-10');
+
+    &:hover,
+    &:focus {
+      @include hb-global-color('color', 'black');
+    }
```

Comment:
Ah! Interesting! These styles were declared for Traditional but not for Colorful. I guess they were not needed until the external link combination issue surfaced.